.gitignore

```diff
@@ -9,3 +9,8 @@ docs/_build/
 .DS_Store
 dist/
 *~
+/venv3/
```

Comment:
These `.gitignore` changes have nothing to do with this change and should be removed.